makefiles/Makefile.common

```diff
@@ -33,13 +33,55 @@ CUDART=-lcudart
 CURAND=-lcurand
 HIPRAND=-lhiprand
 
-COMPILER_AVAILABLE := clang gnu intel icx intel_win mic pgi cray xl hip xlp8 aocc
-
-ifeq ($(wildcard $(OPS_INSTALL_PATH)/../makefiles/Makefile.$(OPS_COMPILER)),)
-print:
-	@echo "unrecognised value for OPS_COMPILER: "$(OPS_COMPILER)
+ifeq ($(OPS_COMPILER),clang)
```

Comment:
Let's not bring this back. Revert to the old one, unless there is a good reason

Reply:
the latest makefile doesnt have this. maybe ur looking it before rebase??

Reply:
https://github.com/OP-DSL/OPS/blob/703b27a3330cd4c79e78c31434f5f641dfb47218/makefiles/Makefile.common#L36
I am saying we should not have the ugly ifeq structure, instead the wildcard stuff

Reply:
ok corrected it